app/index.js

```diff
@@ -25,7 +25,7 @@ var versions = {
   'master': 'master',
   '0.4.0': 'v0.4.0',
   '0.4.1': 'v0.4.1',
-  '0.4.2': 'v0.4.2'
+  '0.4.2': '0.4.2'
```

Comment:
@lirantal Did you post a link to the 0.4.2 tag anywhere? If not, let's delete it.